workflow/Snakefile

```diff
@@ -21,8 +21,12 @@ include: "rules/quant.smk"
 include: "rules/diffexp.smk"
 include: "rules/diffsplice.smk"
 include: "rules/enrichment.smk"
+include: "rules/datavzrd.smk"
 
 
 rule all:
     input:
         all_input,
+        "results/QC/QC_plot.html" 
+        if config["experiment"]["3-prime-rna-seq"]["activate"]
+        else all_input,
```

Comment:
Rather than handling here, I'd add this to all_input at the place where that is generated.

Reply:
Noted. Done